cmd/crates/soroban-spec-typescript/src/lib.rs

```diff
@@ -96,7 +96,12 @@ pub fn generate(spec: &[ScSpecEntry]) -> String {
             cases: vec![],
         });
     }
-    let (fns, other): (Vec<_>, Vec<_>) = collected
+    // Filter out function entries with names that start with "__"
+    let filtered_collected: Vec<_> = collected
+        .into_iter()
+        .filter(|entry| !matches!(entry, Entry::Function { name, .. } if name.starts_with("__")))
+        .collect();
```

Comment:
You don't need to collect since you immediately convert back into an iterator.

Reply:
good call